pkgs/tools/system/illum/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchgit, bash, pkgconfig, ninja, libevdev, libev }:
+
+stdenv.mkDerivation rec {
+  version = "0.4";
+  name = "illum-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/jmesmon/illum.git";
+    fetchSubmodules = true;
+    rev = "48ce8631346b1c88a901a8e4fa5fa7e8ffe8e418";
+    sha256 = "05v3hz7n6b1mlhc6zqijblh1vpl0ja1y8y0lafw7mjdz03wxhfdb";
+  };
+
+  buildInputs = [ bash pkgconfig ninja libevdev libev ];
+
+  configurePhase = ''
+    bash ./configure
```

Comment:
Hm, I see. Sounds like an impurity issue (if travis runs on ubuntu, `/bin/sh` is `dash`, I think). You could either patch the shebang of `configure` (e.g., using the `patchShebangs` helper in `preConfigure`) or leave it as is; `bash` ought to be available in the build environment without adding it to the build inputs, at any rate.